src/Spectre.libException/ObjectMovedException.h

```diff
@@ -0,0 +1,30 @@
+/*
+* ObjectMovedException.h
+* Thrown when using object invalidated by move operation.
+*
+Copyright 2017 Grzegorz Mrukwa
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include "ExceptionBase.h"
+
+namespace Spectre::libException
+{
+class ObjectMovedException: public ExceptionBase
+{
+public:
+    ObjectMovedException();
```

Comment:
This was used in one of the previous versions of this branch. Now is completely out of date and removed.